en/mkdocs.yml

```diff
@@ -275,13 +275,13 @@ nav:
                 - Deployment Options:
                     - Overview: deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-deploy-overview.md
                     - Installation Prerequisites: deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/installation-prerequisites.md
-                    - Choreo Connect on Docker:
+                    - On Docker:
                         - With WSO2 API Manager as a Control Plane : deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-docker-with-apim-as-control-plane.md
                         - As a Standalone Gateway: deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-as-a-standalone-gateway-on-docker.md
-                    - Choreo Connect on Kubernetes - Helm Artifacts:
+                    - On Kubernetes with Helm Artifacts:
                         - With WSO2 API Manager as a Control Plane: deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane-helm-artifacts.md
                         - As a Standalone Gateway: deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-as-a-standalone-gateway-on-kubernetes-helm-artifacts.md
-                    - Choreo Connect on Kubernetes - YAML Artifacts:
+                    - On Kubernetes with YAML Artifacts:
                         - With WSO2 API Manager as a Control Plane: deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane.md
                         - As a Standalone Gateway: deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-as-a-standalone-gateway-on-kubernetes.md
                 - Endpoints:
```

Comment:
This change is to make this
<img src="https://user-images.githubusercontent.com/17605554/161994668-44805ce7-8a1f-4716-a6e5-fa741a9d8c35.png" width="180" height="250">
look like this
<img src="https://user-images.githubusercontent.com/17605554/161994776-c7cf2ee3-0a1b-40f1-8617-d5b4ac20c807.png" width="180" height="200">